introduction.md

```diff
@@ -1 +1,7 @@
-Hello, world. I'm Lindsey. I <3 HTML & CSS. 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?